### ISSUE TITLE
Screen Recorder OS Crash

### ISSUE THREAD

**Issue by JohnnyC1234** [PixelExperience/android-issues#1764]
## Build date

Example: PixelExperience_begonia-12.0-20220121-0746-OFFICIAL.zip

## Expected Behavior

The integrated screen recorder app should work 

## Current Behavior

The phone freezes

## Steps to Reproduce

1. Swipe down twice from the top edge of your screen to pull down the settings menu
2. There select the Screen Recorder 
3. Start the recording, there is a 3 seconds countdown, after which it starts recording and then the phone freezes

/device begonia
/version twelve

**Comment by pixelexperiencebot** [comment]
Issue created! You can close at any time by commenting ```/close```

**Comment by ZIDAN44** [comment]
Till now no fix for it

**Comment by pixelexperiencebot** [comment]
Issue closed by maintainer request.

**Comment by JohnnyC1234** [comment]
Yea thanks, just noticed that this was posted here before. Wanted to say that I tried several apps most of them caused the crash or they just stoped but this one actually works "XRecorder" https://play.google.com/store/apps/detailsid=videoeditor.videorecorder.screenrecorder&hl=en&gl=US


I also posted on the forum it contains two tips I would add/change for the official installation guide if you have time to look at it. https://forum.xda-developers.com/t/rom-12-begonia-begoniain-pixel-experience-official-aosp.4196041/post-86449935

Thanks you for your work.
